### PR TITLE
Correct typo in sample code for range-based for loop.

### PR DESCRIPTION
**Pull request by FredTingaud**

### CODE REVIEW

**Comment by FredTingaud** [comment]
I signed it!

**Comment by FredTingaud** [comment]
CLA signing website refuses to use the correct Google account, so I guess we'll have to wait for it to be corrected before I can sign for this pull request ¯\_(ツ)_/¯

**Comment by dominichamon** [comment]
Looks like the commit was made with a gotsai.com email. Perhaps if you amend the commit author to the one you signed it will work?

**Comment by FredTingaud** [comment]
Thanks @dominichamon! I was so focused on the CLA displaying the wrong address (it only displays the 
"main" when I wanted to sign with the secondary), that I didn't see I had the wrong address on my side too.

File: README.md [hunk 1]
```diff
@@ -200,7 +200,7 @@ In C++11 mode, a ranged-based for loop should be used in preference to
 the `KeepRunning` loop for running the benchmarks. For example:
 
 ```c++
-static void BM_Faste(benchmark::State &st) {
+static void BM_Faste(benchmark::State &state) {
   for (auto _ : state) {
```
Comment (dominichamon) [inline README.md]: Can you rename the bm to "fast" too while you're here please?

**Comment by FredTingaud** [comment]
@dominichamon done for the other typo too.

**Pull request merged**